doc/packer.txt

```diff
@@ -345,6 +350,8 @@ q                    close the display window
 <CR>                 toggle information about a particular plugin
 r                    revert an update
 
+They can be configured by changing the value of `config.display.keybindings`.
```

Comment:
Perhaps we should describe what values this config variable can take? Presumably either a table like in the example or (maybe) the value `false` to disable all keybindings?